microservices/app-builder-menu-bff/src/client/requestConfig.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express';
+import { AxiosRequestConfig } from 'axios';
+
+export function getRequestConfig(req: Request): AxiosRequestConfig<any> {
+
+    const requestConfig: AxiosRequestConfig<any> = {}
+
+    // Forward Authorization header
+    const authHeader = req.header('Authorization')
+    if (authHeader !== undefined) {
+        requestConfig.headers = { Authorization: authHeader }
+    }
+
+    return requestConfig
+}
```

Comment:
if you want, here you can prevent the `any` inference type in this way:

```javascript
import { Request } from 'express';
import { AxiosRequestConfig, AxiosRequestHeaders } from 'axios';

type RequestConfig = AxiosRequestConfig<{
    headers?: AxiosRequestHeaders
}>

export function getRequestConfig(req: Request) {
  const requestConfig: RequestConfig = {};

  // Forward Authorization header
  const authHeader = req.header('Authorization');
  if (authHeader !== undefined) {
    requestConfig.headers = { Authorization: authHeader };
  }

  return requestConfig;
}
```

In this way, when you inspect the function return you have a predictable type.

Reply:
Done, thanks